src/feb.c

```diff
@@ -1464,6 +1468,31 @@ int API_FUNC qthread_readFE(aligned_t *restrict       dest,
     return QTHREAD_SUCCESS;
 }                      /*}}} */
 
+/* the way this works is that:
+ * 1 - src's FEB state is ignored
+ * 2 - data is copied from src to destination
+ */
+
+int API_FUNC qthread_readXX(aligned_t *restrict       dest,
+                            const aligned_t *restrict src)
+{                      /*{{{ */
+    qthread_t          *me      = qthread_internal_self();
+
+    assert(qthread_library_initialized);
+
+    if (!me) {
+        return qthread_feb_blocker_func(dest, (void *)src, READXX);
```

Comment:
The only reason I had it use the blocker_func was so that it was guaranteed to run in qthread, otherwise the shepherd passed to qthread_debug and UNIQUERECORD could be null. 

Maybe that's not a good enough reason, in which case it's easy to remove from the debug call, and maybe we don't need to call UNIQUERECORD for readXX anyways